### PR TITLE
Feat : add Card and carousel

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: Components/card.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    font-size: 1.25rem;
+}
+
+.bat-card button {
+    padding: 0.5rem 1rem;
+}
+
+.bat-card img {
+    max-width: 100%;
+    width: 300px;
```
Comment (Hamdrive) [inline Components/card.css:56]: Why both max-width and width is used?

File: Components/slider.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    display: inline-block;
+    width: 15px;
+    height: 15px;
+    border-radius: 50%;
+    background: rgba(255, 255, 255, 0.5);
+    cursor: pointer;
+}
+
+.active {
+    background: #ffff !important;
```
Comment (Hamdrive) [inline Components/slider.css]: Neccesaary to use `!important`?

File: docs/card.html [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+              <button><a href="./badge.html">Badge</a></button>
+            </li>
+            <li>
+              <button><a href="./button.html">Button</a></button>
+            </li>
+            <li>
+              <button class="bat-active-btn">
+                <a href="./card.html">Card</a>
+              </button>
+            </li>
```
Comment (Hamdrive) [inline docs/card.html:82]: Show other list elements like this

File: style.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+@import url('./docs/css/index.css');
+@import url('./Components/modal.css');
+@import url('./Components/navbar.css');
+@import url("./Components/toast.css");
+@import url('./Components/slider.css');
+@import url('./Components/grid.css');
+@import url('./Components/list.css');
+@import url('./Components/rating.css');
+@import url('./Components/typography.css');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```
Comment (Hamdrive) [inline style.css]: Double import, in same file, can be removed

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Excellent code structure and great naming practices followed
Suggestions have been mentioned, also fixes

Also suggestion to have a proper title and description for PR

Noticed that alert, avatar CSS files are also part of this PR, 
Suggest to remove those as it doesnt match with title

File: Components/card.css (hunk 1 above)
Comment (ZuberDunge) [reply Components/card.css:56]: Because, I wanted adjust it according to screen with % but also don't go beyond 300px

File: style.css (hunk 4 above)
Comment (ZuberDunge) [reply style.css]: Thanks, done.

File: Components/slider.css (hunk 2 above)
Comment (ZuberDunge) [reply Components/slider.css]: Oh that was a mistake I'll remove that

File: Components/card.css (hunk 1 above)
Comment (Hamdrive) [reply Components/card.css:56]: Understood, then approved

File: Components/slider.css (hunk 2 above)
Comment (Hamdrive) [reply Components/slider.css]: Done

**Review by Hamdrive — COMMENTED** [review]
All changes look good, approved and can be merged

**Comment by rahul0396** [comment]
Whole code is clean now and its look good and work good. so approved from my side

**Review by Hamdrive — APPROVED** [review]
Approved changes

**Review by rahul0396 — APPROVED** [review]
Whole code is clean now and its look good and work good. so approved from my side.

**Pull request merged**